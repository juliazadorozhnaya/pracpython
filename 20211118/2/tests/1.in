class C:
    num = Num()
 
print(C().num)
c, d = C(), C()
a = 1234567
c.num = a
d.num = str(a)
print(c.num, d.num)
b = eval("5**3")
c.num = b
d.num = str(b)
print(c.num, d.num)
