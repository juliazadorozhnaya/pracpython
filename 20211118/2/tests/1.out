0
1234567 7
125 3
